tools/sgcollect_info

```diff
@@ -111,16 +111,16 @@ def make_http_client_pprof_tasks(sg_url):
         "goroutine",
     ]
 
-    base_pprof_url = "{}/_debug/pprof".format(sg_url)
+    base_pprof_url = "{0}/_debug/pprof".format(sg_url)
```

Comment:
Can you add a link to a reference for the change to explicit indexes (?), since this is 99% of the changes here it would be useful to understand.